index.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
-function createError(msg, code, props) {
-    var err = msg instanceof Error ? msg : new Error(msg);
+function createError(err, code, props) {
     var key;
 
+    if (!(err instanceof Error)) {
+        throw createError(new Error('Please pass an Error to err-code'), 'ERR_INVALID_ARGUMENTS');
```

Comment:
Perhaps throw a TypeError instead of a Error? Also, I think the error codee is not necessary as the TypeError is what is common to use when invalid arguments are passed.

Reply:
I thought it'd be good to dog-food.  I've changed it to be a TypeError.